app/resources/messages.py

```diff
@@ -125,7 +125,7 @@ def post(self):
         is_draft = False
         draft_id = None
         if 'msg_id' in post_data:
-            is_draft = DraftById().check_valid_draft(post_data['msg_id'])
+            is_draft = DraftModifyById().check_valid_draft(post_data['msg_id'])
```

Comment:
Is this the best place to be checking whether something is a draft as it is common functionality for a number of endpoints?